Makefile

```diff
@@ -139,7 +142,22 @@ TMPDIR ?= $(if $(shell if [ -d /scratch ] ; then echo true; fi),/scratch/$(USER)
 
 ### Default tag
 help:
-	@echo "Targets:"
+	@echo
+	@echo "This is the primary Makefile for both an archive file based setup"
+	@echo "(i.e. created from daikon-<version>.zip) and for a repository"
+	@echo "based setup (i.e. git cloned from //github.com/codespecs/daikon)."
+	@echo "This is primarily for users who wish to customize or extend DAIKON."
```

Comment:
It's also for anyone who wants to use any tools together with Daikon, such as DynComp, right?
Can this paragraph be cut?